app/assets/stylesheets/components/_super-navigation-header.scss

```diff
@@ -1,16 +1,33 @@
 $breakpoint: 769px;
+$header-height: 55px;
 
 .app-c-super-navigation-header,
 .app-c-super-navigation-header__container {
   border-bottom: 0;
   margin-bottom: 0;
   padding-top: 0;
+  z-index: 4;
+}
+
+.app-c-super-navigation-header__container.govuk-width-container {
+  margin-left: 0;
+  margin-right: 0;
+
+  @include govuk-media-query($from: $breakpoint) {
+    margin-left: auto;
+    margin-right: auto;
+  }
 }
 
 .app-c-super-navigation-header__header-logo {
   padding-top: govuk-spacing(2) + 4;
```

Comment:
Can you add a comment to explain this magic number?
```
padding-top: govuk-spacing(2) + 4;
```

Reply:
Better yet, assign to a variable with a name that means what it does